jetbrains-ultimate/src/software/aws/toolkits/jetbrains/services/lambda/go/GoSamDebugSupport.kt

```diff
@@ -3,10 +3,31 @@
 
 package software.aws.toolkits.jetbrains.services.lambda.go
 
+import com.intellij.execution.process.ProcessAdapter
+import com.intellij.execution.process.ProcessEvent
 import com.intellij.execution.runners.ExecutionEnvironment
+import com.intellij.openapi.application.ExpirableExecutor
+import com.intellij.openapi.application.impl.coroutineDispatchingContext
+import com.intellij.openapi.application.runInEdt
+import com.intellij.openapi.util.registry.Registry
+import com.intellij.util.concurrency.AppExecutorUtil
+import com.intellij.xdebugger.XDebugProcess
 import com.intellij.xdebugger.XDebugProcessStarter
+import com.intellij.xdebugger.XDebugSession
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import org.jetbrains.concurrency.AsyncPromise
+import org.jetbrains.concurrency.Promise
+import software.amazon.awssdk.services.lambda.model.PackageType
+import software.amazon.awssdk.services.lambda.model.Runtime
+import software.aws.toolkits.core.utils.getLogger
+import software.aws.toolkits.core.utils.warn
+import software.aws.toolkits.jetbrains.core.utils.buildList
 import software.aws.toolkits.jetbrains.services.lambda.execution.sam.SamDebugSupport
 import software.aws.toolkits.jetbrains.services.lambda.execution.sam.SamRunningState
+import software.aws.toolkits.jetbrains.utils.ApplicationThreadPoolScope
+import java.net.InetSocketAddress
+import java.nio.file.Files
 
 class GoSamDebugSupport : SamDebugSupport {
```

Comment:
Should we override isSupported to be false (and update it to a min sam version once its patched?)

Reply:
I was going to wait for it to be fixed to merge go support, we can decide what we want to do when we get closer to release though